src/ray/common/task/scheduling_resources.cc

```diff
@@ -686,6 +694,13 @@ void ResourceIdSet::AddOrUpdateResource(const std::string &resource_name,
   }
 }
 
+void ResourceIdSet::CommitImplicityBundleResourceIds(const PlacementGroupID &group_id,
```

Comment:
Same here. It doesn't seem to be returned?

Reply:
same as above.